tests/common/include/tests/common/core_detector.inl

```diff
@@ -608,10 +608,10 @@ TEST(detector, detector_volume_construction) {
               sf_finder_links[geo_obj_id::e_passive]);
 
     EXPECT_EQ(d.surfaces().size(), 19u);
-    EXPECT_EQ(d.mask_store().template size<mask_id::e_portal_cylinder2>(), 3u);
+    EXPECT_EQ(d.mask_store().template size<mask_id::e_portal_cylinder2>(), 2u);
```

Comment:
Why did numbers change here?

Reply:
Because the beampipe is intersected as a passive cylinder surface now, which can have two solutions (when you propagate from the outside in). It is using the general cylinder intersector instead of the one that's optimized to portals now, which means the mask shape has a different type and needs to go into a different container

Reply:
OK